src/main/java/com/commercetools/sync/categories/utils/CategoryUpdateActionUtils.java

```diff
@@ -4,204 +4,219 @@
 import com.commercetools.sync.categories.CategorySyncOptions;
 import io.sphere.sdk.categories.Category;
 import io.sphere.sdk.categories.CategoryDraft;
-import io.sphere.sdk.categories.commands.updateactions.*;
+import io.sphere.sdk.categories.commands.updateactions.ChangeName;
+import io.sphere.sdk.categories.commands.updateactions.ChangeSlug;
+import io.sphere.sdk.categories.commands.updateactions.SetDescription;
+import io.sphere.sdk.categories.commands.updateactions.ChangeParent;
+import io.sphere.sdk.categories.commands.updateactions.ChangeOrderHint;
+import io.sphere.sdk.categories.commands.updateactions.SetMetaTitle;
+import io.sphere.sdk.categories.commands.updateactions.SetMetaDescription;
+import io.sphere.sdk.categories.commands.updateactions.SetMetaKeywords;
 import io.sphere.sdk.commands.UpdateAction;
 import io.sphere.sdk.models.LocalizedString;
 import io.sphere.sdk.models.Reference;
 
 import javax.annotation.Nonnull;
 import java.util.Optional;
 
-import static com.commercetools.sync.commons.enums.Warning.*;
+import static com.commercetools.sync.commons.enums.Warning.CATEGORY_CHANGE_ORDER_HINT_EMPTY_ORDERHINT;
+import static com.commercetools.sync.commons.enums.Warning.CATEGORY_CHANGE_PARENT_EMPTY_PARENT;
+import static com.commercetools.sync.commons.enums.Warning.CATEGORY_SET_DESCRIPTION_EMPTY_DESCRIPTION;
 import static com.commercetools.sync.commons.utils.CommonTypeUpdateActionUtils.buildUpdateAction;
 
 public final class CategoryUpdateActionUtils {
 
     /**
      * Compares the {@link LocalizedString} names of a {@link Category} and a {@link CategoryDraft} and returns an
-     * {@link UpdateAction<Category>} as a result in an {@link Optional}. If no update action is needed, for example in
-     * case where both the {@link Category} and the {@link CategoryDraft} have the same name, an empty {@link Optional}
-     * is returned.
+     * {@link UpdateAction&lt;Category&gt;} as a result in an {@link Optional}. If no update action is needed,
+     * for example in case where both the {@link Category} and the {@link CategoryDraft} have the same name,
+     * an empty {@link Optional} is returned.
      *
      * @param oldCategory the category which should be updated.
      * @param newCategory the category draft where we get the new name.
      * @return A filled optional with the update action or an empty optional if the names are identical.
      */
     @Nonnull
     public static Optional<UpdateAction<Category>> buildChangeNameUpdateAction(
-            @Nonnull final Category oldCategory,
-            @Nonnull final CategoryDraft newCategory) {
+      @Nonnull final Category oldCategory,
+      @Nonnull final CategoryDraft newCategory) {
         final LocalizedString newCategoryName = newCategory.getName();
         return buildUpdateAction(oldCategory.getName(),
-                newCategoryName,
-                () -> ChangeName.of(newCategoryName));
+          newCategoryName,
```

Comment:
because I am using the `newCategorySlug` twice as opposed  to the `oldCategory` which is used once. So I store `newCategorySlug` in a variable and reuse it. I meant it as a kind of a micro optimisation. Actually the compiler will do this optimization on its own. But if it's confusing, then I would change it.